plugin/trino-iceberg/pom.xml

```diff
@@ -340,6 +346,13 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>net.snowflake</groupId>
+            <artifactId>snowflake-jdbc</artifactId>
+            <version>3.14.5</version>
```

Comment:
Move the one in `trino-snowflake` to the root `dependencyManagement` and remove the version here